src/components/AddItem.jsx

```diff
@@ -64,7 +71,19 @@ const AddItem = ({ onNewItem }) => {
       >
         Add
       </button>
+
+      
+
+    </div>
+
+    <div className="m_error">
+        <p className="m_errorMsg" > {errorMsg} </p>
```

Comment:
the error message should be rendered conditionally. If there is Error then only it should be render at particular place.